BUILD.gn

```diff
@@ -59,11 +60,6 @@ if (current_toolchain != "${dir_pw_toolchain}/dummy:dummy") {
       "${nlunit_test_root}:nlunit-test",
     ]
 
-    # Temporary until RNG issue sorted out
```

Comment:
is this part of the echo work?  if not, would you mind putting this in a separate PR?

Reply:
Yes, we need to unlink the old wml based message stack to avoid name conflict.